### PR TITLE
Update countries

### PR DESCRIPTION
**Pull request by promag**

### CODE REVIEW

File: data/countries.json [hunk 1]
```diff
@@ -2810,12 +2816,23 @@
...
-    "independent": "Part of FI",
+    "independent": "No",
     "name": {
       "short": "Åland Islands",
       "uppercase": "ÅLAND ISLANDS"
     },
     "numeric": "248",
     "status": "officially-assigned"
+  },
+  {
```
Comment (promag) [inline data/countries.json]: @ruimarinho considering only countries with status `officially-assigned` and `transitionally-reserved`this is missing in current version.

File: data/countries.json [hunk 2]
```diff
@@ -867,15 +869,15 @@
     "status": "officially-assigned"
   },
   {
-    "alpha2": "TF",
+    "alpha2": "FQ",
```
Comment (promag) [inline data/countries.json]: @ruimarinho This was wrong.

File: data/countries.json [hunk 3]
```diff
@@ -867,15 +869,15 @@
...
     "alpha3": "ATF",
-    "independent": "Territory of FR",
+    "independent": "",
     "name": {
-      "short": "French Southern Territories",
-      "uppercase": "FRENCH SOUTHERN TERRITORIES"
+      "short": "French Southern and Antarctic Territories",
+      "uppercase": "FRENCH SOUTHERN AND ANTARCTIC TERRITORIES"
     },
-    "numeric": "260",
```
Comment (promag) [inline data/countries.json:877]: There is no `numeric` for this country.

**Comment by promag** [comment]
Note that 

```
  {
    "alpha2": "XK",
    "alpha3": "XKX",
    "name": {
      "short": "Kosovo",
      "uppercase": "KOSOVO"
    }
  }
```

does not exist in the ISO.

**Comment by ruimarinho** [comment]
@promag I am aware of that, but since this temporary ISO code is already widely used, I want to keep it.

**Comment by ruimarinho** [comment]
Thank you @promag!

**Pull request merged**